### ISSUE TITLE
Fix 4 Maintainability issues in sheet_id\utils\BilinearUpSampling.py

### ISSUE THREAD

**Issue by mjenrungrot** [mjenrungrot/SheetIdentification#1]
[CodeFactor](https://www.codefactor.io/repository/github/mjenrungrot/sheetidentification/overview/master) found multiple issues:

#### Trailing newlines
[sheet_id\utils\BilinearUpSampling.py:93
](https://www.codefactor.io/repository/github/mjenrungrot/sheetidentification/source/master/sheet_id/utils/BilinearUpSampling.py#L93)
#### Unused argument 'mask'
[sheet_id\utils\BilinearUpSampling.py:83
](https://www.codefactor.io/repository/github/mjenrungrot/sheetidentification/source/master/sheet_id/utils/BilinearUpSampling.py#L83)
#### Wildcard import keras.layers
[sheet_id\utils\BilinearUpSampling.py:3
](https://www.codefactor.io/repository/github/mjenrungrot/sheetidentification/source/master/sheet_id/utils/BilinearUpSampling.py#L3)
#### Unnecessary "else" after "return"
[sheet_id\utils\BilinearUpSampling.py:84
](https://www.codefactor.io/repository/github/mjenrungrot/sheetidentification/source/master/sheet_id/utils/BilinearUpSampling.py#L84)
